src/NServiceBus.Core.Tests/Pipeline/PipelineModelBuilderTests.cs

```diff
@@ -109,8 +163,7 @@ public void ShouldDetectMissingBehaviorForRootContext()
             {
                 RegisterStep.Create("Child", typeof(ChildBehaviorOfChildContext), "desc"),
 
-            }, new List<RemoveStep>(), new List<ReplaceStep>());
-
+            }, new List<RemoveStep>(), new List<ReplaceStep>(), new List<AddOrReplaceStep>());
```

Comment:
I wonder if we could add some test helper method to create the PipelineModelBuilder to be able to skip newing up all the empty lists? (for readability purposes)